cpp/src/plasma/test/client_tests.cc

```diff
@@ -503,7 +503,7 @@ TEST_F(TestPlasmaStore, RefreshLRUTest) {
 TEST_F(TestPlasmaStore, DeleteTest) {
   ObjectID object_id = random_object_id();
 
-  // Test for deleting non-existence object.
+  // Test for deleting nonexistent object.
```

Comment:
Why these changes?

Reply:
The word "nonexistence" is a noun, so I guess "nonexistence object" means an object that represents the state of nonexistence.
On the other hand, the word "nonexistent" is an adjective, so I guess "nonexistent object" means an object that is absent.

Reply:
Ah, it didn't know it was actually a word. Thank you :-)